esphome/components/nextion/nextion_base.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+#include "esphome/core/defines.h"
+#include "esphome/core/color.h"
+#include "nextion_component_base.h"
+namespace esphome {
+namespace nextion {
+
+#define USE_TFT_UPLOAD
+
+//#define NEXTION_PROTOCOL_LOG  // For testing purposes
+
+#ifdef ESPHOME_LOG_HAS_VERY_VERBOSE
+#define NEXTION_PROTOCOL_LOG
+#endif
+
+#ifdef NEXTION_PROTOCOL_LOG
+#ifdef ESPHOME_LOG_HAS_VERY_VERBOSE
+#define ESP_LOGN(tag, ...) esph_log_vv(tag, __VA_ARGS__)
```

Comment:
```suggestion
#define ESP_LOGN(tag, ...) ESP_LOGVV(tag, __VA_ARGS__)
```

I believe that should work (maybe with small modifications). Anyway, integrations shouldn't use `esph_log_*` functions directly as we only want to expose the `ESP_LOG*` macros.

Reply:
I dont mind removing it. I kept it in to help with debugging the protocol and queue handling. Nice to see you!

@ssieb and I have been working on this. I believe he has a review he is working on.